ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,4621 @@
+            )
+        except CommandFailed as exc:
+            if "ENOENT" in str(exc):
+                log.warning(f"Ceph auth entity {entity} not found")
+                return ""
+            raise
+        if isinstance(result, dict):
+            return result.get("key", "")
+        return str(result).strip()
+
+    @staticmethod
+    def log_auth_key_snapshot(label, keys):
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `log_auth_key_snapshot`

Add return type annotation: `None`

(ANN205)